src/commands/command.ts

```diff
@@ -16,6 +16,7 @@ export interface CommandPermission {
 
 export interface CommandBase {
     readonly name: string,
+    readonly aliases?: String[],
```

Comment:
`string[]` pls

Reply:
da bekommt ich nen fehler weil es später mit String type verglichen wird
daher String[]

Reply:
Argument of type 'String' is not assignable to parameter of type 'string'

Reply:
wat? ok dann lass so